esmvalcore/_recipe.py

```diff
@@ -836,12 +825,25 @@ def _update_timerange(variable, config_user):
     timerange = variable.get('timerange')
     check.valid_time_selection(timerange)
 
-    timerange = _format_years(timerange)
-    check.valid_time_selection(timerange)
-
     if '*' in timerange:
         (files, _, _) = _find_input_files(variable, config_user['rootpath'],
                                           config_user['drs'])
+        if not files:
+            if not config_user.get('offline', True):
+                msg = (
+                    " Please note that automatic download is not supported "
```

Comment:
Is the initial space intentional?
```suggestion
                    "Please note that automatic download is not supported "
```

Reply:
Nope, this needs to stay like this to get a nice formatting. Note that there is no space after the full stop here:`f"time range '{timerange}'.{msg}"`.